frontend/components/common/BasePanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <modal :visible="true" :mask-closable="false" :closable="false" :footer="null">
```

Comment:
Why did you use a modal? The issue and the mock doesn't say anything about the modal 🙂